FiducialRegistrationWizard/MRML/vtkMRMLFiducialRegistrationWizardNode.cxx

```diff
@@ -319,6 +320,56 @@ int vtkMRMLFiducialRegistrationWizardNode::UpdateModeFromString( std::string upd
 }
 
 //------------------------------------------------------------------------------
+void vtkMRMLFiducialRegistrationWizardNode::SetInputFormat( int newInputFormat )
+{
+  if ( this->GetInputFormat() == newInputFormat )
+  {
+    // no change
+    return;
+  }
+  this->InputFormat = newInputFormat;
+  this->Modified();
+  this->InvokeCustomModifiedEvent(InputDataModifiedEvent);
+}
+
+//------------------------------------------------------------------------------
+std::string vtkMRMLFiducialRegistrationWizardNode::InputFormatAsString( int inputFormat )
+{
+  switch ( inputFormat )
+  {
+    case INPUT_FORMAT_ORDERED_PAIRS:
+    {
+      return "Ordered Pairs";
+    }
+    case INPUT_FORMAT_UNORDERED_PAIRS:
+    {
+      return "Unordered Pairs";
+    }
+    default:
+    {
+      vtkGenericWarningMacro( "Unrecognized value for InputFormat " << inputFormat << ". Returning \"Unknown\"" );
+      return "Unknown";
+    }
+  }
+}
+
+//------------------------------------------------------------------------------
+int vtkMRMLFiducialRegistrationWizardNode::InputFormatFromString( std::string inputFormatAsString )
```

Comment:
InputFormat is not a very expressive name. Something like PointMatchingMethod would be better.